external/fv3fit/tests/emulation/test_data_io.py

```diff
@@ -1,57 +0,0 @@
-import os
-import tempfile
-import numpy as np
-import xarray as xr
-
-from fv3fit.emulation.data import io
-
-
-def _get_ds():
-
-    return xr.Dataset(
-        {
-            "air_temperature": xr.DataArray(
-                data=np.arange(30).reshape(10, 3), dims=["sample", "z"]
-            ),
-            "specific_humidity": xr.DataArray(
-                data=np.arange(30, 60).reshape(10, 3), dims=["sample", "z"]
-            ),
-        }
-    )
-
-
-def test_get_nc_files():
```

Comment:
the integration test I added in test_netcdf covers these cases...and the get_nc_files function is basically a thin wrapper around `glob`...so probably not worth unit testing in this way.